tests/src/cli.js

```diff
@@ -4,82 +4,144 @@
 import path from 'path';
 
 import { expect } from 'chai';
-import { CLIEngine } from 'eslint';
+import { CLIEngine, ESLint } from 'eslint';
 import eslintPkg from 'eslint/package.json';
 import semver from 'semver';
+import * as importPlugin from '../../src/index';
 
 describe('CLI regression tests', function () {
   describe('issue #210', function () {
+    let eslint;
     let cli;
     before(function () {
-      cli = new CLIEngine({
-        useEslintrc: false,
-        configFile: './tests/files/issue210.config.js',
-        rulePaths: ['./src/rules'],
-        rules: {
-          'named': 2,
-        },
-      });
+      if(ESLint) {
+        eslint = new ESLint({
+          useEslintrc: false,
+          overrideConfigFile: './tests/files/issue210.config.js',
+          rulePaths: ['./src/rules'],
+          overrideConfig: {
+            rules: {
+              'named': 2,
+            },
+          },
+          plugins: { 'eslint-plugin-import': importPlugin },
+        });
+      } else {
+        cli = new CLIEngine({
+          useEslintrc: false,
+          configFile: './tests/files/issue210.config.js',
+          rulePaths: ['./src/rules'],
+          rules: {
+            'named': 2,
+          },
+        });
+        cli.addPlugin('eslint-plugin-import', importPlugin);
```

Comment:
why is this needed now, but wasn't needed before?

Reply:
If we don't specify this, I think eslint-plugin-import in node_modules will be used for linting.
I think I need this change to check if my PR change does not affect this test.

Reply:
ok, so it seems like this is a change we could/should add separately, since it will fix the issue of tests running with the last published version, instead of the current code?

Reply:
I will open another PR later.

Reply:
I opened PR #2197